code/game/objects/items/boxcutter.dm

```diff
@@ -46,5 +50,8 @@
 	on = active
 	playsound(src, on_sound, 50)
 	tool_behaviour = (active ? TOOL_KNIFE : NONE)
+	if(tool_behaviour == TOOL_KNIFE)
+		AddElement(/datum/element/cuffsnapping, snap_time_weak_handcuffs, snap_time_strong_handcuffs)
+	else
+		RemoveElement(/datum/element/cuffsnapping, snap_time_weak_handcuffs, snap_time_strong_handcuffs)
```

Comment:
why are you checking the tool behaviour instead of whether or not its active

Reply:
didnt know we had that!

Reply:
what do you mean didnt know. its literally set in the same proc